### DISCUSSION TITLE
Sort order not working?

### DISCUSSION THREAD

**Discussion by PatrickA3** [LycheeOrg/Lychee#1836] [Q&A] [answered]
Hello,

I am trying to change the sort order of subalbums in an album (in the about screen of the album change the order by) and this doesn't seem to work. It doesn't matter if I select a Title or Description and change Ascending or Descending it always keep being ordered in the way the global setting is set (Title, Ascending).

When I change the global setting for albums to Title, Descending all the subalbums also change in sort order.

Is there a way to keep the global album sort order on Title, Descending and the sub albums sort order on Title Ascending?

**Comment by ildyria** [comment]
Unfortunately not at the moment. That is something that would be really useful to add I agree.

**Reply by PatrickA3** [reply]
That too bad. When I was searching for an answer it looked like the same issue as this one #319 so I was hoping this would work.

**Reply by ildyria** [reply, marked as answer]
In theory the ordering of sub albums should follow the main ordering, if that is not the case, then it is a problem.
What I would be more interested to see added (by any volunteer) is that similar to pictures in an album, it would be possible to sort sub albums accordingly per album.

For now only photo ordering is possible per album.

**Reply by PatrickA3** [reply]
You are right, my sub albums follow the ordering of the main albums. So it works as designed.
In my use case I would need the main albums to sort descending and the subalbums ascending, so I would need exactly the option you are describing. Hopefully someone will add the option. Thanks for clarifying!

**Reply by ildyria** [reply]
I asked a friend to see if he can work on it, but he is new to php, so that may take some time. :)
Fortunately most of the tools to implement it are already there.
Needs to be done:
- [ ] : add settings per album to take over default behavior when set.
- [ ] : Use Sorting decorator over the Collection of children.
- [ ] : Add UI to allow to change such settings
- [ ] : Add API end point (with validation) to allow to change such settings.

**Reply by ildyria** [reply]
See here: https://github.com/LycheeOrg/Lychee/issues/1358
